reference/forms/types/options/choice_loader.rst.inc

```diff
@@ -0,0 +1,74 @@
+choice_loader
+~~~~~~~~~~~~~
+
+**type**: :class:`Symfony\\Component\\Form\\ChoiceList\\Loader\\ChoiceLoaderInterface`
+
+The ``choice_loader`` can be used to only partially load the choices in cases where
+a fully-loaded list is not necessary. This is only needed in advanced cases and
```

Comment:
I think we should get rid of "advanced cases" as nobody knows what that means. Is it possible to define the use cases broadly here instead?

Reply:
Thanks for the feedback! I've pushed a rewording.